cypress/integration/phase.spec.js

```diff
@@ -34,8 +35,8 @@ describe('Phase functionality', () => {
     navigateToForm();
     const formValues = {
       phaseName: 'Test phase',
-      startDate: '2021/03/30',
-      endDate: '2022/03/31',
+      startDate: '1990/01/01',
+      endDate: '1992/01/01',
```

Comment:
What's the reason for this change?

Reply:
As a fresh start, I went through and added all new dates to the phase tests to ensure there were no overlaps. It likely could have stayed as it was though.